client/signup/jetpack-connect/index.jsx

```diff
@@ -133,57 +162,38 @@ const JetpackConnectMain = React.createClass( {
 		} else {
 			this.checkUrl( this.state.currentUrl );
```

Comment:
@johnHackworth It looks like this was introduced in 91d218b9e7efa7fd840430f15b0feb1c835625b9, do you recall why `isRequestingSites` is relevant for `checkUrl`?

See https://github.com/Automattic/wp-calypso/pull/13368/files/e81db2ecca73005d2c37c359af579c6bc70af38a#diff-e2b84252aef88e8d9da883e8f7ac5e78